packages/flutter/lib/src/widgets/keyboard_listener.dart

```diff
@@ -53,6 +53,9 @@ class KeyboardListener extends StatelessWidget {
   });
 
   /// Controls whether this widget has keyboard focus.
+  ///
+  /// The owner will be responsible for calling [FocusNode.dispose] on the focus
+  /// node when it is done with it.
   final FocusNode focusNode;
```

Comment:
Why the comment here and below is removed?
It is saying that the owner, i.e the code that created the object, should dispose it, that is correct.

Reply:
I am not sure what you mean by that.
For explanation: I *added* the documentation because the class didn't clarify that the owner is still responsible for object disposal. This lack of documentation made the leak occur in the first place and created the possibility of error.

Reply:
Got it. Sorry for confusion.

It is a common convention that owner should dispose an object. If object x obtained object y as a method argument, it does not make x owning y, so x is not responsible for the disposal.

This should not be explained, because if we will explain it, this explanation will be needed for every disposable argument.
Makes sense?
So, I think the comment should be removed.